docs/development-setup/prerequisites.md

```diff
@@ -90,24 +90,15 @@ Core:
 
 * [Fortanix Rust EDP] utilities.
 
-  Make sure a _nightly_ version of the Rust toolchain is installed:
-
-  ```
-  rustup install nightly
-  ```
-
-  Then install the Fortanix Rust EDP utilities by running:
+  Install the Fortanix Rust EDP utilities by running:
 
   <!-- markdownlint-disable line-length -->
   ```
-  cargo +nightly install fortanix-sgx-tools
-  cargo +nightly install sgxs-tools
+  cargo install fortanix-sgx-tools
+  cargo install sgxs-tools
   ```
   <!-- markdownlint-enable line-length -->
 
-  _NOTE: These utilities must be compiled with a nightly version of the Rust
-  toolchain since they use the `#![feature]` macro._
-
 * Oasis Core's Rust toolchain version with Fortanix SGX target.
```

Comment:
Do we need to remove/update this section also, as nightly is not needed anymore?